public/utils/alerting.ts

```diff
@@ -48,14 +48,15 @@ export const createIndexPatterns = async (
       dataSourceRef,
     });
   } catch (err) {
-    if (err instanceof DuplicateIndexPatternError) {
+    console.error('Create index pattern error', err.message);
+    // Err instanceof DuplicateIndexPatternError is not a trusted validation in some cases, so find index pattern directly.
```

Comment:
out of courisity ,do we have an example of other errors instead of DuplicateIndexPatternError?

Reply:
Normally no, because we will create the index pattern according to the agreed parameters, but I guess unless there is a case where the timeFieldName is passed incorrectly, this is the situation we discussed on before that needs to  deep dive alerting DSL mechanism.